Disable game mode voting over a certain cap (#5495)
tgstation/TerraGov-Marine-Corps: config/config.txt

## Comment this out to disable checking for the cid randomizer dll. (disabled if database isn't enabled or connected)
CHECK_RANDOMIZER

> ##Set this number to however max players you want after which game mode votes cannot be started

Reviewer: ```suggestion
##Set this number to the maximum number of players you want, after which game mode votes cannot be started
```
